addons/stock_account/models/product.py

```diff
@@ -207,7 +203,12 @@ def _change_standard_price(self, old_price):
         return
 
     def _get_standard_price_at_date(self, date=None):
+        """ Get Last Price History """
         self.ensure_one()
+        if not date or date == fields.Date.today():
```

Comment:
Can't comment below, but the `if date:` later on doesn't make sense anymore, since now if there's no `date` we already return the standard_price